plugins/11_TheWarWithin/zones/undermine.lua

```diff
@@ -711,6 +804,171 @@ map.nodes[67482929] = ScrapHeap({criteriaID = 103187}) -- Blackwater Marina
 
 ns.hook.Vignette({group = ns.groups.SCRAP_REWARDS, vignetteID = 6687})
 ns.hook.Vignette({group = ns.groups.SCRAP_REWARDS, vignetteID = 6757})
+ns.hook.Vignette({group = ns.groups.SCRAP_REWARDS, vignetteID = 6700}) -- Gallagio Garbage
+
+-------------------------------------------------------------------------------
+------------------------ SHIPPING AND HANDLING REWARDS ------------------------
+-------------------------------------------------------------------------------
+
+local ShippingHandling = Class('ShippingHandling', Collectible, {
+    label = '{spell:1222389}', -- Shipping & Handling: Job Streak
+    icon = 132764,
+    scale = 1.5,
+    rewards = {
+        Pet({item = 232847, id = 4646}), -- Personal-Use Sapper (86303)
+        Pet({item = 232852, id = 4636}), -- Mutt (86302)
+        Mount({item = 229947, id = 2303}), -- Violet Goblin Shredder (86300)
+        Transmog({item = 231556, slot = _G.CHESTSLOT}) -- Breakneck Cabbie's Coat (86299)
+    }
+}) -- Shipping & Handling 10/10 rewards
+
+function ShippingHandling.getters:note()
+    return C_Spell.GetSpellDescription(1222389)
+end
+
+map.nodes[15852496] = ShippingHandling()
```

Comment:
`map.nodes[90003500] = ShippingHandling()`

better pos, some addons will cover it.